### DISCUSSION TITLE
"task-spooler" package points to a fork

### DISCUSSION THREAD

**Discussion by viric** [termux/termux-packages#19445] [General]
Hello,

I'm the copyright owner task-spooler. I see that @twaik once updated it to a fork:
https://github.com/termux/termux-packages/commit/8dd857fe16eba5d637280750d34129165c1b04b0

The person of that fork decided to publish his own new versions of task-spooler with the same name, instead of rebranding the derived GPL work. I asked that person to stop publishing task-spooler as if it were his own: https://github.com/justanhduc/task-spooler/issues/56

If you use the name 'task-spooler' for the package, please point to the main source (as it was previous to @twaik commit) instead of that fork. If you prefer that fork, please pick a different name.

**Comment by sylirre** [comment]
~~@viric Would it be acceptable to change package name to `task-spooler-ng`?~~

Reverted to original and updated to v1.0.3. Hopefully this issue now resolved at Termux side.

**Comment by viric** [comment]
I don't like that the person said task spooler was not maintained. It's something he decided and then took over. As "-ng" sounds to me pretending to take over as well. He mainly added some gpu-related things; maybe task-spooler-gpu? No idea. I hope he answered with a new brand name, but maybe he abandoned his fork.

**Reply by sylirre** [reply]
I haven't really looked into the difference between original and the fork. If main feature of "v2.0" is presence of gpu support (disabled during build time), we can just stick to the original task-spooler.

**Reply by sylirre** [reply]
Reverted to the original: https://github.com/termux/termux-packages/commit/2ad9edd4079a8ed158e2eda41e675dc0e2619b3a

**Comment by viric** [comment]
He started publishing versions of ts even before 2.0. No idea. I only recently noticed. I think the whole point of the fork was the 'gpu' thing, since the beginning. Not only 2.0.

**Comment by viric** [comment]
fwiw I just released a bugfix version, 1.0.3.
https://viric.name/soft/ts/
